contracts/core/price-aggregator/src/events.rs

```diff
@@ -13,18 +13,27 @@ pub struct NewRoundEvent<M: ManagedTypeApi> {
     epoch: u64,
 }
 
+#[type_abi]
+#[derive(TopEncode)]
+pub struct DiscardSubmissionEvent {
+    submission_timestamp: u64,
```

Comment:
Have timestamp, epoch, block as different types. Improves code readability by a lot, instead of having just "u64" everywhere.
i.e.
`pub type Timestamp = u64;`, same for others